vktrace/vktrace_common/vktrace_interconnect.c

```diff
@@ -125,7 +125,7 @@ BOOL vktrace_MessageStream_SetupHostSocket(MessageStream* pStream) {
 #if defined(PLATFORM_LINUX) || defined(PLATFORM_OSX)
     int yes = 1;
 #endif
-    struct addrinfo hostAddrInfo = {0};
+    struct addrinfo hostAddrInfo = {};
```

Comment:
{0} was meant to clear up the struct members. Should we call memset 0 to stay on the safe side?